README.md

```diff
@@ -1 +1,66 @@
-# International-Writers-Conference-2021
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project - International-Writers-Conference-2021
+
+This is a Microverse Capstone project for HTML, CSS and Javascript DOM. I set up my Capstone Project about an imaginary International Writers' Conference in Athens, Greece. The goal is to make 2 pages (home and about) for mobile and desktop version.
+
+- ![screenshot](./Assets/mobile-home.png)
+- ![screenshot](./Assets/desktop-home.png)
+- ![screenshot](./Assets/mobile-about.png)
+- ![screenshot](/Assets/desktop-about.png)
+
+## :red_circle: Project Documentation Video
+
+[Project Video](https://www.loom.com/share/730101c6b2f940fc881ed57e87b42527)
+## :red_circle: Live Demo
+
+[Live Preview Link](https://gunjuzone.github.io/International-Writers-Conference-2021/)
+
+## :hammer: Built With
+
+- HTML, CSS and JavaScript
+
+To get a local copy up and running follow these simple steps:
+
+1. Go to the [repository page](https://github.com/Gunjuzone/International-Writers-Conference-2021/tree/mobile-version).
+2. Press the "Code" button and copy the link.
+3. Clone it using git command `git clone <link>`.

```

Comment:
Let's be more helpful for non-technical users so that they can set up your app on a local machine easily.

Example: 
- git clone `your repo`
- cd into the folder
- git checkout branchName
- open `index.html` with live server